updating 2.2 troubleshoot for mco

dockerymick opened the pull request:
Revision of 2.2. after review of 2.1 @clyang82 can you help confirm that my changes are accurate for 2.2?

https://github.com/open-cluster-management/backlog/issues/8303

dockerymick committed:
updating 2.2 troubleshoot for mco

clyang82 commented on troubleshooting/trouble_obs_mco.adoc:
```diff
@@ -1,38 +1,60 @@
 == Symptom: PVC does not change after editing statefulSetSize
 
-After you update the `statefulSetSize`, you might receive an alert that reads, `KubePersistentVolumeFillingUp`. The `data-observability-observatorium-thanos-receive-default-x` PVCs might be affected.
+After you update the `statefulSetSize`, you might receive an alert that reads, `KubePersistentVolumeFillingUp`. The `data-observability-observatorium-thanos-receive-default-*` PVCs might be affected.
```
`you might receive an alert that reads,` I think we do not need to have `comma` in the end.

clyang82 commented on troubleshooting/trouble_obs_mco.adoc:
```diff
@@ -1,38 +1,60 @@
 
-. Expand the `data-observability-observatorium-thanos-receive-default-x` PVC to reflect the edited `statefulSetSize` in the MultiClusterObservability (mco) CR. Run the following command:
+. Expand the `data-observability-observatorium-thanos-receive-default-*` PVC to update the `storage` parameter to a greater value than the `statefulSetSize` value  from the MultiClusterObservability (mco) CR. Run the following command for each `data-observability-observatorium-thanos-receive-default-*` PVC:
 +
```
Here has a format issue so that the whole line is bold.

dockerymick replied:
Thanks for pointing this out, that's interesting bc the previous example of the PV name has the same syntax. I will continue to use `x` instead of the asterisk.

clyang82 commented on troubleshooting/trouble_obs_mco.adoc:
```diff
@@ -1,38 +1,60 @@
 +
 ----
-kubectl get pvc data-observability-observatorium-thanos-receive-default-0 -o yaml
+kubectl get pvc data-observability-observatorium-thanos-receive-default-* -o yaml
```
cannot run the command with `*`. suggest to keep using `0` here.

dockerymick replied:
Thanks for clarifying

dockerymick committed:
Update trouble_obs_mco.adoc

Syntax error (inconsistent) when there was an asterisk surrounded by code tick (`) make the variable monospaced.
Along with ensuring that variables are italic monospaced according to the IBM Style Guide.
Changes after 2nd review

dockerymick commented:
@clyang82 I have made updates based on your feedback. May you re-review when you have an opportunity?

clyang82 reviewed (approved):
lgtm
